api/server/structs/endpoints_beacon.go

```diff
@@ -134,10 +134,10 @@ type GetBlockAttestationsResponse struct {
 }
 
 type GetBlockAttestationsV2Response struct {
-	Version             string      `json:"version"`
-	ExecutionOptimistic bool        `json:"execution_optimistic"`
-	Finalized           bool        `json:"finalized"`
-	Data                interface{} `json:"data"` // Accepts both `Attestation` and `AttestationElectra` types
+	Version             string            `json:"version"`
+	ExecutionOptimistic bool              `json:"execution_optimistic"`
+	Finalized           bool              `json:"finalized"`
+	Data                []json.RawMessage `json:"data"` // Accepts both `Attestation` and `AttestationElectra` types
```

Comment:
If you make this a single `json.RawMessage`, you can call `json.Marshal` just once on the entire slice.